cubiesMoves.py

```diff
@@ -0,0 +1,97 @@
+import sys
```

Comment:
You're not using this file?

Seems very interesting but I don't understand anything